docs/issuer/on-chain-issuer/on-chain-tutorial.md

```diff
@@ -12,44 +12,42 @@ keywords:
   - smart contract
 ---
 
-This guide presents an example of how to use the **on-chain issuer**. We created a small application, where we communicate with Metamask to retrieve the user's balance and a claim about this balance is generated via the on-chain issuer.
+Currently, we have two approaches to issuing on-chain credentials: **on-chain merklized issuer** and **on-chain non-merklized issuer**.
+The difference between these two approaches:
+1. The **on-chain merklized issuer** calculates the Merkle root for a credential on the backend and stores the core claim of the credential on-chain using a smart contract. We can think of it as a centralized issuer with the ability to store trees on the blockchain.
```

Comment:
, because merklization process is too expensive to do in a smart contract by the user. Credential data and metadata can stay private on issuer side and only hash of the credential will be published on chain.